### PR TITLE
initial quiz submission

### PR DESCRIPTION
**Pull request by markeasterling**

### CODE REVIEW

**Comment by gregkorte** [comment]
Hey Mark. Looks like you just have the styling left on your quiz. Let me know when you finish, and I'll take another look.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="/quiz.css">
```
Comment (gregkorte) [inline index.html]: This just needs to be changed to reflect the name of your css file, and you can leave off the `/`.

**Comment by gregkorte** [comment]
Hey Mark. See that one change, and you are good to merge after that. :+1:

**Comment by markeasterling** [comment]
thanks, greg! I'm working on the javascript quiz right now.

**Pull request merged**